addons/mail/static/src/models/emoji_list_view/emoji_list_view.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'EmojiListView',
+    identifyingFields: ['popoverViewOwner'],
+    recordMethods: {
+        onClickEmoji(ev) {
+            if (this.popoverViewOwner.messageActionListOwnerAsReaction) {
+                this.popoverViewOwner.messageActionListOwnerAsReaction.message.addReaction(ev.currentTarget.dataset.unicode);
+            }
+            if (this.popoverViewOwner.composerViewOwnerAsEmoji) {
+                this.popoverViewOwner.composerViewOwnerAsEmoji.insertIntoTextInput(ev.currentTarget.dataset.unicode);
+                if (!this.messaging.device.isMobileDevice) {
+                    this.popoverViewOwner.composerViewOwnerAsEmoji.update({ doFocus: true });
+                }
+            }
+            if (this.popoverViewOwner) {
```

Comment:
Guard after the field has been accessed above? One of the methods has deleting it as side effect?